tests/services/document-fetcher.test.js

```diff
@@ -84,7 +84,10 @@ describe('DocumentFetcher', () => {
     const expectedPredicatesAtValue = Symbol('AT')
     const uid = Symbol('UID')
     const prismicApi = {
-      query: jest.fn(() => ({ results: [expectedValue] })),
+      query: jest
+        .fn()
+        .mockImplementationOnce(() => ({ results: [expectedValue] }))
+        .mockImplementationOnce(() => ({ results: [] })),
```

Comment:
question : Pourquoi a-t-on besoin d'avoir deux fois un mock ici ?

Reply:
On fait deux appels `query`: un dans `getSimplePageByUid`et un autre dans`getSlicesPageByUid`.
Le premier `mockImplementationOnce` retourne le résultat du premier appel, et le 2e `mockImplementationOnce` retorune le résultat du 2e appel

Reply:
Ça ne me semble pas évident. Peut-être qu'on y gagnerait à sortir tout ça dans des fonctions ?

Reply:
@sbedeau sortir quelle partie?

Reply:
Pardon pour le manque de clarté.

Ça donnerait quelque chose comme ça je pense : 
```suggestion
        .mockImplementationOnce(getSimplePageByUidMock)
        .mockImplementationOnce(getSlicesPageByUidMock),
```

Reply:
done =)